md/installation_guide.md

```diff
@@ -0,0 +1,238 @@
+    |---- installation-guide.md                   // this file
+    |---- bin/bonita-ici                          // an utility that start the dockers and deploy L.A.s
+    |---- bin/bonita-ici.bat                      // an utility that start the dockers and deploy L.A.s (Windows)
+    |---- jdbc_drivers/                           // the folder where to put the jdbc driver of the Bonita platform database
+```
+
+
+### Elasticsearch
+
+ICI server requires elasticsearch. Supported version is 6.2.3 or above, in cluster or single node mode. 
+
+For more information, please refer to the [official installation guide](https://www.elastic.co/downloads/elasticsearch)
+
+
+### ICI server
+
+1. Copy ici-application-<VERSION>.zip` to the host. 
+2. Unzip it in the directory of your choice. 
+
+We will refer to the extracted directory as `ICI_APPLICATION_HOME`.
+
+#### Configuration
+
+Edit `$ICI_APPLICATION_HOME/application.properties` to configure application.
+Mandatory parameters are:
+
+In Elastic search configuration
+```
+bonita.elasticsearch.hosts              // http url(s) of installed Elasticsearch 
+
+# when elastic search endpoint are protected by authentication,
+# rest client basic authentication can be enabled by adding properties as follow
+#bonita.elasticsearch.username=admin
+#bonita.elasticsearch.password=changeme
+```
+
+In Bonita database configuration
+```
+bonita.datasource.url                   // url of Bonita platform database
+bonita.datasource.username              // username used to connect to Bonita database (can be a read-only user)
+bonita.datasource.password              // password associated to the user used to connect to Bonita database
+```
+
+Oracle only: copy Jdbc driver in `$ICI_APPLICATION_HOME/lib` (drivers for PostgreSQL, MySQL and MS SqlServer are already provided).
+
+#### Advanced configuration
+
+Other parameters are available in `$ICI_APPLICATION_HOME/application.properties`.  
+We recommend you to read this file and change other parameters if needed.
+
+#### Advanced polling profile mode (Oracle only)
+
+This mode requires that database user is allowed to create materialized views. To grant this, use 
+`GRANT CREATE MATERIALIZED VIEW TO <USER>` using a SYS connection prior to start application.  
+
+To activate this mode, uncomment this property in `$ICI_APPLICATION_HOME/application.properties`
+
+```
+#bonita.ici.polling.profile=advanced
+```
+
+### Launch
+
+To launch the application, simply run `$ICI_APPLICATION_HOME/bin/bonita-ici` on Linux 
+or `$ICI_APPLICATION_HOME/bin/bonita-ici.bat` on Windows.
+
+This will start a web server on port 8082 by default (if not changed in `$ICI_APPLICATION_HOME/application.properties`)
+
+### Installation of ICI Living Applications
+
+#### Using the deployer
+
+This method is the recommended way. Use command below, you will be prompted for required parameters
+
+```shell
+./bonita-ici deploy
+```
+
+#### Manually
+
+This method is the not recommended way.
```

Comment:
So why documenting it? 
If it has other advantages, could you list them?

Reply:
this was the only way prior to have a LA deployer. in Race condition (profile name already existing), Patty or Dave may want to customize the LA and install it step by step

we kept this because it's a fall back, id LA deployer fails for timeout or other unknown issues

this is present in v1.0.0 install guide distribution